### DISCUSSION TITLE
请问启动时有参数(配置)可以指定工作目录吗

### DISCUSSION THREAD

**Discussion by zjkwdy** [Mrs4s/go-cqhttp#773] [Q&A]
请问启动时有参数(配置)可以指定工作目录吗，类似
`go-cqhttp --work-dir /etc/qbot/`
或者有其他方式实现定义工作目录的方法吗？

**Comment by Ink-33** [comment]
目前没有，您可能需要在工作目录运行您的gocq（即使gocq不在工作目录也能正常工作）
